libafl/src/executors/inprocess/mod.rs

```diff
@@ -90,16 +90,24 @@ where
         input: &I,
     ) -> Result<ExitKind, Error> {
         *state.executions_mut() += 1;
+
+        // unsafe operations for settings up pointers
```

Comment:
Always good to add a
```rust
// # Safety
// <why this is safe> 
```
annotation.
Makes it easier for reviewer (and easier for me to import this into our codebase..)